### PR TITLE
feat: toggleTheme

### PR DESCRIPTION
**Pull request by saurabbot**
## What does this PR do?
This pr adds a feature which allows the user the app theme.
Implements #9047 

https://www.loom.com/share/c3f222b633954baeb6305ec61e3f133d

## How should this be tested?

- [x] Test A->changing the themes and checking the change in db
- [ ] Test B

### CODE REVIEW

File: apps/web/pages/settings/my-account/appearance.tsx [hunk 1]
```diff
@@ -85,45 +87,82 @@ const AppearanceView = () => {
   if (!user) return null;
 
   const isDisabled = isSubmitting || !isDirty;
+  console.log(user.appTheme, formMethods, "user");
 
   return (
     <Form
       form={formMethods}
       handleSubmit={(values) => {
+        console.log(values, "updatemutate");
```
Comment (leog) [inline apps/web/pages/settings/my-account/appearance.tsx:96]: This should be removed

File: packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ type UpdateProfileOptions = {
 
 export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions) => {
   const { user } = ctx;
+  console.log(input, "updateinput");
```
Comment (leog) [inline packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts:29]: This should be removed

File: packages/ui/components/form/inputs/Input.tsx [hunk 3]
```diff
@@ -338,7 +338,7 @@ type FormProps<T extends object> = { form: UseFormReturn<T>; handleSubmit: Submi
 
 const PlainForm = <T extends FieldValues>(props: FormProps<T>, ref: Ref<HTMLFormElement>) => {
   const { form, handleSubmit, ...passThrough } = props;
-
+  console.log(form, "ww");
```
Comment (leog) [inline packages/ui/components/form/inputs/Input.tsx:341]: This should be removed

**Review by leog — CHANGES_REQUESTED** [review]
Thanks for the contribution @firfox341. It does look good! Left some comments regarding code to be removed.

Also, what would be the rationale behind using a new column in the database versus the existing `theme` column?

**Comment by leog** [comment]
Makes sense @firfox341. I would like to invoke the presence of @Jaibles for this one. I'm not sure we want to maintain both variables at the same time; if you choose dark mode why would you like your booking page to be light mode or automatic? Just in case, his blessing is always a good thing to have 😄

**Comment by Jaibles** [comment]
@leog  @saurabbot is the question: "Should we allow people to have different dark/light mode settings for app.cal.com vs their public booking page"? If so, the answer is definitely yes. 

I think a perfectly likely scenario is people who prefer dark mode wanting that in the app, but wanting to go with system settings for the public page so that their bookers get to see what they prefer.

**Comment by Jaibles** [comment]
Settings - My Account - Appearance

We do need to use the new svgs though. I haven't had a chance to optimise them yet in terms of filesize.

File: packages/prisma/.env
Comment (zomars) [inline packages/prisma/.env:1]: Why was this file deleted? Let's put it back:

`git checkout origin/main packages/prisma/.env`

File: packages/ui/components/form/inputs/Input.tsx [hunk 4]
```diff
@@ -338,7 +338,6 @@ type FormProps<T extends object> = { form: UseFormReturn<T>; handleSubmit: Submi
 
 const PlainForm = <T extends FieldValues>(props: FormProps<T>, ref: Ref<HTMLFormElement>) => {
   const { form, handleSubmit, ...passThrough } = props;
-
```
Comment (zomars) [inline packages/ui/components/form/inputs/Input.tsx:341]: Unnecessary diff here. Let's revert it.

File: apps/web/pages/settings/my-account/appearance.tsx [hunk 5]
```diff
@@ -85,6 +87,7 @@ const AppearanceView = () => {
   if (!user) return null;
 
   const isDisabled = isSubmitting || !isDirty;
+  console.log(user.appTheme, formMethods, "user");
```
Comment (zomars) [inline apps/web/pages/settings/my-account/appearance.tsx:90]:
```suggestion
```

File: packages/features/settings/ThemeLabel.tsx [hunk 6]
```diff
@@ -4,20 +4,20 @@ interface ThemeLabelProps {
   label: string;
   defaultChecked?: boolean;
   register: any;
+  page: string;
 }
 
 export default function ThemeLabel(props: ThemeLabelProps) {
   const { variant, label, value, defaultChecked, register } = props;
-
```
Comment (zomars) [inline packages/features/settings/ThemeLabel.tsx:11]: Missing padding here

**Review by zomars — CHANGES_REQUESTED** [review]
Thank you for your contribution 🙏 Left some comments to be addressed before merging

**Comment by Jaibles** [comment]
> gtg right ? is the yarn.lock change fine ? could you guys merge these prs also #8975 #8925

@saurabbot I'm assuming the new svgs I made aren't in yet so that should be blocking. I'll prepare them now.

**Comment by Udit-takkar** [comment]
@saurabbot any update on this PR?

**Comment by leog** [comment]
> Hey @Udit-takkar this was done long back and ready from my side, I was waiting for you guys to test the feature.

Hey @saurabbot. Actually, we were waiting for you to resolve the comments we did in the code (addressed already) and request a re-review so it returns to us to review again to be able to approve it. Take that into account for future contributions, so we can merge great contributions like yours faster 👍

**Comment by leog** [comment]
> @leog gtg i guess..

@firefox341 did you tested the latest? There is some problem with loading the svgs and when saving, the toast message does not look good. Please test it locally again and tackle those issues.

**Comment by leog** [comment]
@saurabbot are you able to continue working on this PR? we might need to close it if there is no more activity in it 😓

**Pull request closed without merging**